docs/modules/a2c.rst

```diff
@@ -47,6 +47,8 @@ Dict          ❌     ✔️
 Example
 -------
 
+Examples are here only to show how to quickly run something. Optimized hyperparameters and proper training are done in the RL Zoo repository found `here <https://github.com/DLR-RM/rl-baselines3-zoo>`_.
```

Comment:
This could use a bit more gentler wording, such as:

> This example is only to demonstrate the use of the library and its functions, and the trained agents may not solve the environments. Optimized hyperparameters can be found in RL Zoo repository `here <https://github.com/DLR-RM/rl-baselines3-zoo>`_.